recipe/gen_patch_json.py

```diff
@@ -527,6 +528,17 @@ def _gen_new_index(repodata, subdir):
         if record_name == "iris":
             _rename_dependency(fn, record, "nc_time_axis", "nc-time-axis")
 
+        # google-api-core-grpc==1.20.0 omitted an update to grpcio>=1.29.0
+        # https://github.com/conda-forge/google-api-core-feedstock/pull/43
+        if record_name == 'google-api-core-grpc':
+            pversion = pkg_resources.parse_version(record['version'])
+            v1_18_0 = pkg_resources.parse_version('1.18.0')
+            v1_20_0 = pkg_resources.parse_version('1.20.0')
+            if v1_18_0 <= pversion <= v1_20_0:
+                with suppress(ValueError):
```

Comment:
You can use 

```
if 'depends' in record and 'grpcio >=1.8.2' in record['depends']:
```

instead of catching a very generic error here.

Reply:
the suggested method iterates over the dependencies twice. `ValueError` is the defined interface of `index` and reduces "copy&paste" errors between the condition and the index, so I'm not sure it's a clear win.

Reply:
It is in code clarity. Please change it.